zone/zone.h

```diff
@@ -99,7 +99,8 @@ class Zone {
 	AA::Ability *GetAlternateAdvancementAbilityByRank(int rank_id);
 	AA::Rank *GetAlternateAdvancementRank(int rank_id);
 	bool is_zone_time_localized;
-	bool process_mobs_while_empty;
+	bool idle_when_empty;
+	uint32 seconds_before_idle;
```

Comment:
We really should try and make these private, we can access them within the Zone class but outside such as `questmanager` we'll need a get/set method - easy to generate in Clion